doc/api/events.markdown

```diff
@@ -137,10 +137,17 @@ Execute each of the listeners in order with the supplied arguments.
 Returns `true` if event had listeners, `false` otherwise.
 
 
+### emitter.listenerCount(type)
+
+* `type` {String} The type of event
```

Comment:
But we have a check now, which expects the `type` to be string. Is that okay to change this to `Value`?